tools/Dockerfile

```diff
@@ -0,0 +1,5 @@
+# Dockerfile for the KDC image
+
+FROM nvaziri/kdc:dev_0
```

Comment:
Is this Dockerfile ever built? Doesn't look like this tag exists: https://hub.docker.com/r/nvaziri/kdc/tags/